### ISSUE TITLE
Adding Script for All Image Blurring techniques

### ISSUE THREAD

**Issue by Ashishsingh619** [geekquad/Image-Processing-OpenCV#245]
### I want to add a Ipython script to demonstrate all Image blurring techniques like:
- Averaging Blur
-  Gaussian Blur
-  Median Blur
- Bilateral Filtering
@geekquad Sir and @unnati914 ma'am please assign me with this issue.

**Comment by geekquad** [comment]
Hey @Ashishsingh619, isn't it the same as filtering?

**Comment by Ashishsingh619** [comment]
Yes @geekquad Sir it is but all issues for filtering techniques are closed, So I wanted to implement this and the issue of averaging Blur Technique is not even raised by any participant. So I wanted to implement them.

**Comment by geekquad** [comment]
Go ahead with averaging Blur Technique for now. 
Let me have a look at the remaining others and I will let you know.
